### ISSUE TITLE
Feature request: `novel_recommended` method

### ISSUE THREAD

**Issue by book000** [upbit/pixivpy#248]
Hello, Thanks for developing and maintaining a great library.

Currently, [`illust_recommended`](https://github.com/upbit/pixivpy/blob/cdd5b8a74d2614fd398eca745b788d647118524a/pixivpy3/aapi.py#L283) is the method to retrieve recommended illustrations.
However, there is no corresponding method for novels.
I propose adding a method like `novel_recommended`.

I did some research on my own and it seems that I can get the expected data with the `https://app-api.pixiv.net/v1/novel/recommended` endpoint.
In practice, the following simple code could be used to obtain it.

```python
def novel_recommended(api: AppPixivAPI,
                      include_ranking_label=True):
    url = '%s/v1/novel/recommended' % api.hosts

    params = {
        'include_ranking_label': api.format_bool(include_ranking_label),
        'filter': 'for_ios',
    }

    r = api.no_auth_requests_call('GET', url, params=params, req_auth=True)
    return api.parse_result(r)
```

However, this is a simplified version for my own use, and it will be necessary to check parameters to actually incorporate it as a library.

I look forward to reply.

**Comment by book000** [comment]
## Endpoint parameters

The parameters that can be checked from `next_url` are listed for reference.

- `include_ranking_label` = `true`
- `filter` = `for_ios`
- `include_ranking_novels` = `false`
- `offset` = `15`
- `already_recommended` = `11111111,22222222` (comma separated novel ids)
- `max_bookmark_id_for_recommend` = `2001285579`
- `include_privacy_policy` = `false`

**Comment by upbit** [comment]
> ## Endpoint parameters
> The parameters that can be checked from `next_url` are listed for reference.
> 
> * `include_ranking_label` = `true`
> * `filter` = `for_ios`
> * `include_ranking_novels` = `false`
> * `offset` = `15`
> * `already_recommended` = `11111111,22222222` (comma separated novel ids)
> * `max_bookmark_id_for_recommend` = `2001285579`
> * `include_privacy_policy` = `false`

Good job, let me verify these parameters

**Comment by upbit** [comment]
@book000 You can try the new version of master, by `pip install git+https://github.com/upbit/pixivpy.git@master`

I tried it out and it works fine, see [novel_recommended() demo](https://github.com/upbit/pixivpy/blob/e5c7062aa77e34a18058e9e2516b2ea853f9d596/demo.py#L236-L252)

I'am lack of verification environment for the `novel_recommended()`, please help to verify. If there is no problem, let me know and I will tag and release it.

**Comment by book000** [comment]
OK, I have checked it in my environment and it works fine.

Thanks for your fast respond!

**Comment by upbit** [comment]
Fixed in [v3.7.2](https://pypi.org/project/PixivPy3/3.7.2/)
